### ISSUE TITLE
Avoid ableist and gender-specific language

### ISSUE THREAD

**Issue by realeroberto** [immuni-app/immuni-documentation#43]
This expands on #22.

Failure in understanding that public digital services' inclusiveness is an absolute priority is a huge telltale mark.

Contact tracing must be non-discriminatory in every aspect, as stressed multiple times by EU-level policymakers (not earlier than two days ago by the [European Parliament](https://www.europarl.europa.eu/news/en/press-room/20200512IPR78915/covid-19-tracing-apps-meps-stress-the-need-to-preserve-citizens-privacy)), and this includes a number of topics such as accessibility (#8), multilingual support (#13), privacy by design even in the documentation (#16), using a gender-neutral language (#22), and avoiding ableisms.

I will go to such lengths as to claim that a failure in delivering a thoroughly inclusive digital or service is tantamount to a failure of the whole project, as it would pave the way for a less fair society. And that is true regardless of whether the service were elsewise successful in fighting the pandemics or not.

Come on: inclusiveness is no unattainable goal even with such hectic time constraints as we all are facing. A good starting point are Google's [guidelines on writing inclusive documentation](https://developers.google.com/style/inclusive-documentation).

@eutopian-eu

**Comment by xb8** [comment]
#23 can probably be closed aswell for the same reasons

**Comment by ALBUCHERQUE** [comment]
LOL now whe have also the LGTBQ guy *facepalm*
